src/Ztm.Zcoin.NBitcoin.Tests/ZcoinTransactionData.cs

```diff
@@ -0,0 +1,35 @@
+namespace Ztm.Zcoin.NBitcoin.Tests
+{
+    static class ZcoinTransactionData
+    {
+        /// <summary>
+        /// Mainnet Coinbase transaction
+        /// </summary>
+        public const string CoinBase =
```

Comment:
Better to use `readonly` instead of `const` due to it is long string.

Reply:
I use it because I need to add it to `InlineData` tag.